server/utils/helpers/articleControllerHelper.js

```diff
@@ -13,6 +22,32 @@ const createArticleResponse = (article, username, bio, image, slug) => ({
     image,
   },
 });
+
+/**
+ * Abstract the included fields on getting article(s)
+ * @param {Object} model sequelize model
+ * @param {Object} where selection criteria
+ * @returns {Array} an array of sequelize
+ * association include options
+ */
+const includeAuthor = (model, where) => [{
+  model,
+  as: 'author',
+  attributes: ['username', 'bio', 'image'],
+  where,
+}];
+
+/**
+ * Abstract the attributes requested on getting article(s)
+ * @returns {Array} array of sequelize attributes
+ */
+const articleAttributes = () => [
+  'slug', 'title', 'description',
```

Comment:
Pls, include image as part of the articles attributes.

Reply:
yeah, @megame24 have you included the `image` as part of the articles attributes ?